messaging-ui.md

```diff
@@ -48,6 +48,9 @@ The AXP Messaging UI provides a highly customizable user interface for messaging
 14. **Edit Images Before Sending**
     Users can edit images before sending them where they can crop, flip and rotate Images, providing a more interactive and personalized messaging experience.
 
+15. **Typing Indicator**
+    Display a typing indicator when the other participant is typing, providing real-time feedback on the conversation status.
```

Comment:
The "other participant" part here seems inaccurate, because it's not always just a two party call. If a supervisor or expert publicly joins the chat, there could be multiple participants on the call center end. Maybe change "the other participant" to "a remote participant".